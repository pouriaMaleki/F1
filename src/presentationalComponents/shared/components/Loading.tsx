import React from "react";
import css from "./Loading.module.sass";
import { PropagateLoader } from "react-spinners";
import { Logo } from "./Logo";

const Loading = () => (
  <div className={css.pageCenter}>
    <Logo className={css.logo} />
    <div className={css.loading}>
      <PropagateLoader
        sizeUnit={"px"}
        size={25}
        color={"#FF2000"}
        loading={true}
      />
    </div>
  </div>
);

export { Loading };
