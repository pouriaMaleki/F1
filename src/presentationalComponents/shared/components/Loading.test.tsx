import React from "react";
import renderer from "react-test-renderer";
import { Loading } from "./Loading";

describe("Render Loading", () => {
  // snapshot test
  it("render correctly Loading component", () => {
    const LoadingComponent = renderer.create(<Loading />).toJSON();
    expect(LoadingComponent).toMatchSnapshot();
  });
});
