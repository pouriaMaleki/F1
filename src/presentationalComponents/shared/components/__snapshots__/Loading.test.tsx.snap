// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render Loading render correctly Loading component 1`] = `
<div
  className="pageCenter"
>
  <img
    alt="f1"
    className="root logo"
    src="logo.png"
  />
  <div
    className="loading"
  >
    <div
      className="css-1eitjko"
    >
      <div
        className="css-s6e7mz"
      />
      <div
        className="css-vtoicm"
      />
      <div
        className="css-m3wc4i"
      />
      <div
        className="css-m0vwee"
      />
      <div
        className="css-o3fj8j"
      />
      <div
        className="css-1q6p993"
      />
    </div>
  </div>
</div>
`;
